docs/kubernetes/kctlr-ingress.rst

```diff
@@ -25,167 +38,159 @@ You can use the |kctlr-long| as an Ingress Controller in Kubernetes.
 
 Unattached pools
-~~~~~~~~~~~~~~~~
+````````````````
 
 You can create :ref:`unattached pools <kctlr-pool-only>` for the Services defined in the Ingress resource. To do so, just omit the ``virtual-server.f5.com/ip=`` annotation from your Ingress resource.
 
 You can then :ref:`use an IPAM system <kctlr-ipam>` to assign an IP address and attach a virtual server to the pools, or use another means to manually route traffic to the pools on the BIG-IP system.
 
-Deployments using multiple Ingress Controllers
-``````````````````````````````````````````````
-
-.. note::
-
-   In Kubernetes, the Ingress resource's :code:`ingress.class` property is unset by default. The |kctlr| automatically manages all Ingress resources that don't have an :code:`ingress.class`` defined.
-
-If you're using more than one Ingress Controller to manage your Ingress resources:
-
-- Set the :code:`ingress.class` property to "f5" for all Ingress resources you want the |kctlr| to manage. ::
-
-   kubernetes.io/ingress.class="f5"
+Initial Setup
+-------------
 
-- Define the :code:`ingress.class` as appropriate for the Ingress resources managed by other Ingress Controllers. The |kctlr| ignores Ingress resources that have any :code:`ingress.class` other than "f5".
+**Skip this step if:**
 
+- If you have already assigned a :ref:`default IP address <ingress default IP>` to the Controller **--OR--**
+- If you are using :ref:`DNS lookup <dns lookup ingress>` to resolve host IP addresses.
 
 .. _ingress self IP:
 
-Initial Setup
--------------
-
-Allocate a `Self IP address`_ from the external network on the BIG-IP system. You'll assign this IP address to the Ingress resource's BIG-IP virtual server.
+Allocate a `Self IP address`_ from the external network on the BIG-IP system; this is the IP address you will assign to the Ingress resource's virtual server.
+If you're running the |kctlr| in :ref:`cluster mode <cluster mode>`, the IP address must be within the subnet associated with the BIG-IP VXLAN tunnel.
 
 .. note::
 
-   - If you intend to create :ref:`unattached pools <kctlr-pool-only>` (pools without a virtual server), you will need to set up another way to route traffic to the pools on the BIG-IP system.
+   If you intend to create :ref:`unattached pools <kctlr-pool-only>` (pools that aren't attached to a virtual server), you will need to set up another way to route traffic to the pools on the BIG-IP system.
 
-   - If you have already assigned a :ref:`default IP address <ingress default IP>` to the Controller, you may skip this step.
+.. _ingress-quick-start:
 
+Annotate existing Ingress Resources using kubectl
+-------------------------------------------------
 
-.. _ingress-quick-start:
+Use :command:`kubectl annotate` to add the supported `Ingress annotations`_ to any existing Ingress resource.
 
-Set Virtual Server Ingress Annotations using kubectl
-----------------------------------------------------
+The examples below demonstrate the correct command line usage.
```

Comment:
We should note that each individual change can be seen by the ctlr, meaning each kubectl execution could cause an update to the BIG-IP in a piecemeal fashion. To make more atomic changes annotations can be added together by appending additional key/value pairs: `kubectl annotation KEY_1=VAL_1 ... KEY_N=VAL_N`